src/axom/slic/docs/sphinx/sections/architecture.rst

```diff
@@ -150,7 +155,7 @@ The list of keywords is summarized in the table below.
 | **<RANK>**          | The MPI rank that emmitted the message. Only applicable|
```

Comment:
```suggestion
| **<RANK>**          | The MPI rank that emitted the message. Only applicable|
```
Please check and fix "emmit" elsewhere as well.

Reply:
Could be referring to Emmitt Smith? 😄